### PR TITLE
[CN] GenStage

### PR DESCRIPTION
**Pull request by Ljzn**
Hi! This is the Chinese version of `gen_stage.md`.

### CODE REVIEW

**Comment by doomspork** [comment]
Thank you and welcome @Ljzn, this is great! 

@cizixs could you take a peek at this when you have a moment?

File: cn/lessons/advanced/gen-stage.md [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+
+虽然我们的例子是1对1的生产者和消费者, 但是在任何一个 stage 都可能拥有多个生产者和多个消费者.
+
+为了更好地说明这些概念, 我们将使用 GenStage 来构建管道, 但首先让我们来探讨一下 GenStage 的角色.
+
+## 消费者与生产者
+
+正如我们所看到的, 我们赋予 stage 的角色很重要. GenStage 的规范中承认三种角色:
+
++ `:producer` — 一个源.  生产者等待消费者的需求并相应消费者的需求.
```
Comment (cizixs) [inline cn/lessons/advanced/gen-stage.md]: typo. 相应 -> 响应

Comment (Ljzn) [reply cn/lessons/advanced/gen-stage.md]: Fixed.

**Comment by cizixs** [comment]
@Ljzn Thanks for the contribution, and I will merge this.

**Comment by cizixs** [comment]
@doomspork I have reviewed and approved this change, but there is still a warning message `Review required: At least one approved review is required by reviewers with write access`. 

What does it mean? Don't I have a write access, or there is more I should do?

**Comment by doomspork** [comment]
Opps, sorry about that @cizixs! @nscyclone is right, I forgot to add that permission to the other team.

On a separate note @nscyclone & @cizixs, do you think it would be helpful to make granular teams for the translators such as `elixirschool/translators/cn`, `elixirschool/translators/jp`, `elixirschool/translators/ru`, etc?

**Comment by nscyclone** [comment]
@doomspork, I think it's a good idea, that might make assignment of reviewers more precise.

**Pull request merged**